src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -164,6 +166,12 @@ public VariantContext calculateGenotypes(final VariantContext vc, final Genotype
             reducedVC = new VariantContextBuilder(vc).alleles(allelesToKeep).genotypes(reducedGenotypes).make();
         }
 
+        //Calculate the expected total length of the PL arrays for this VC to warn the user in the case that they will be exceptionally large
+        final long maxPLLength = GenotypeLikelihoods.numLikelihoods(reducedVC.getNAlleles(), reducedVC.getMaxPloidy(defaultPloidy)) * reducedVC.getNSamples();
```

Comment:
@KevinCLydon I think we're going to want to change this warning to be a bit more conservative, otherwise for large callsets such as GnomAD we'll get warnings all over the place. We should probably just check the size of the PL array for one sample, and warn if that's too big, rather than multiplying that value by the number of samples.

Reply:
That sounds reasonable. Just pushed a new branch with this change